src/main/java/org/jenkinsci/test/acceptance/po/Build.java

```diff
@@ -157,6 +157,14 @@ public String getConsole() {
         return visit(getConsoleTextUrl()).getPageSource();
     }
 
+    public URL getGitBuildDataUrl() {
```

Comment:
These methods should not be part of Build: a helper method in the test case would make more sense.